dao/models.py

```diff
@@ -0,0 +1,14 @@
+from knowledge_graph import db
+
+
+class ZipHazard(db.Model):
+    zip = db.Column(db.Integer, primary_key=True)
+    hazard = db.Column(db.String, primary_key=True)
+    value = db.Column(db.Boolean)
+
+class Hazard(db.Model):
+    hazard_type = db.Column(db.String, primary_key=True)
+    iri = db.Column(db.String)
+    ref_label = db.Column(db.String)
+    description = db.Column(db.String)
+
```

Comment:
Please remove ZipHazard (**lkp_zip_hazards**) and have just **lkp_zip_nodes**. The backend should just need the IRI's, not the hazard_name. We could keep Hazard (**brg_hazards_nodes**) there for reference since the table is small, but it will not be used by the backend.